packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -183,8 +186,11 @@ def log_decorator_wrapper(*args, **kwargs):
                     )
                 except:
                     # log exception if occurs in function
-                    logger_obj.error(
-                        "Exception: %s", sys.exc_info()[1], extra=extra_args
+                    logger_obj.exception(
```

Comment:
I though log.exception already automatically prints the traceback?

Reply:
Yes. I think this is equivalent to ``logger_obj.exception("Exception:")``. Perhaps you can add some extra info on the msg and not just "Exception"

Reply:
My bad forgot to revert this. Also added tests for this part.